### ISSUE TITLE
Even with the exceptions disabled there is still a throw happening

### ISSUE THREAD

**Issue by Hatrickek** [biojppm/rapidyaml#359]
On line 16796 in single header file version; rapidyaml-0.5.0.hpp, there is a throw which causes the header to not compile when exceptions are disabled in the project.
```cpp
explicit constexpr vtable() noexcept :
        invoke_ptr{ [](storage_ptr_t, Args&&...) -> R
            { throw std::bad_function_call(); }
        },
        copy_ptr{ [](storage_ptr_t, storage_ptr_t) noexcept -> void {} },
        move_ptr{ [](storage_ptr_t, storage_ptr_t) noexcept -> void {} },
        destructor_ptr{ [](storage_ptr_t) noexcept -> void {} }
    {}
```

**Comment by biojppm** [comment]
Thanks for reporting. This is coming from an external header in c4core that is not used, and can be safely removed. I will do that soon.

**Comment by biojppm** [comment]
The issue was in c4core, and was fixed in biopjppm/c4core#105. If you look above, you'll see that most of the work was in getting the tests to compile and run cleanly on all the different platform/compiler combinations. Now the same must be done in this repo, so this is not ready yet.
